crates/evm/core/src/constants.rs

```diff
@@ -47,6 +47,17 @@ pub const DEFAULT_CREATE2_DEPLOYER_CODE: &[u8] = &hex!("604580600e600039806000f3
 /// The runtime code of the default CREATE2 deployer.
 pub const DEFAULT_CREATE2_DEPLOYER_RUNTIME_CODE: &[u8] = &hex!("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3");
 
+/// The address that deploys the default CREATE2 deployer contract.
+pub fn get_create2_deployer() -> Address {
```

Comment:
maybe we can use a `static CREATE2_DEPLOYER: LazyLock<Address>` here?

Reply:
good idea